### DISCUSSION TITLE
dotnet is not recognized as an internal or external command  in Azure Devops Pipeline

### DISCUSSION THREAD

**Discussion by Meharaj27** [community/community#78301] [New to GitHub]
I've installed .net sdk 8 in windows server 2019.
Verified the path in system environment variables C:/Program Files/dotnet
verified dotnet version and build is successful when ran through PowerShell and local build in VS is also success.

For a simple console project "HoloWorld", getting this error "dotnet is not recognized as an internal or external command " when ran through azure pipeline, simple pipeline which has" dotnet build --configuration $(buildConfiguration) " command.
Checkout step ,directory step is success and fails at build step.

Please help.

**Comment by queenofcorgis** [comment]
Hi @Meharaj27 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Azure DevOps (https://techcommunity.microsoft.com/t5/azure-devops/bd-p/AzureDevOpsForum). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
